app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
dùng `present?` thay vì `nil?` và nghịch đảo `!` kết quả nhé

phân biệt: present?, nil?, blank? khác nhau thế nào nhé

Reply:
`present?` và `blank?` trả về kết quả đối nghịch nhau.
`nil?` chỉ trả về mil với NilObject, `blank?` trả về `true`với `false`, `empty` và chuỗi gồm toàn khoảng trắng.